openmc/mgxs_library.py

```diff
@@ -1154,8 +1155,8 @@ def set_prompt_nu_fission_mgxs(self, prompt_nu_fission, temperature=294.,
         """
 
         check_type('prompt_nu_fission', prompt_nu_fission,
-                   (openmc.mgxs.PromptNuFissionXS,
-                    openmc.mgxs.PromptNuFissionMatrixXS))
+                   (openmc.mgxs.FissionXS, openmc.mgxs.NuFissionMatrixXS))
+        check_value('prompt', prompt_nu_fission.prompt, [True])
```

Comment:
Ditto here and elsewhere (unless there is a reason for doing this that I am unaware of).